.github/workflows/release.yml

```diff
@@ -22,10 +22,15 @@ jobs:
         uses: actions/setup-go@v1
         with:
           go-version: "1.13.x"
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2020-02-16
```

Comment:
Assume this needs to be kept in sync with other config?

Reply:
Yeah but the same is true for the used Go version.

Reply:
got it